platform-includes/performance/improving-data/native.mdx

```diff
@@ -2,6 +2,7 @@
 You can add Data Attributes to both Spans and Transactions. This data is visible in the trace explorer in Sentry.
 The data must be of type `sentry_value_t`, which can store:
 * 32-bit signed integers,
+* 64-bit signed and unsigned integers,
```

Comment:
Should we mention that 64-bit unsigned values get sent as Strings until further notice? For context, see https://github.com/getsentry/sentry-native/pull/1326#discussion_r2241965997

e.g. add something like
```
<Alert level="warning">

Currently, 64-bit unsigned integers are sent as strings, since they can't be processed as numerical values yet.

</Alert>
```